### ISSUE TITLE
support for 1.20+

### ISSUE THREAD

**Issue by SniffBakaSniff** [tayadev/Deactivate-Portals#1]
Just wonder if you could update this to the latest mc version.

**Comment by tayacrystals** [comment]
totally, I'll take a look at it as soon as i get to it.
it might even just work without much/any changes, but I'll look into it

**Comment by Alexnortung** [comment]
I could not get it to work in 1.21 neither on my server or single player :)

**Comment by tayacrystals** [comment]
yeah, 1.21 changed where the files have to be in the datapack, so I'll have to rearrange them. I'll update you here once i get to it
